cross/armel/toolchain.cmake

```diff
@@ -16,7 +16,7 @@ add_compile_options(--sysroot=${CROSS_ROOTFS})
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --sysroot=${CROSS_ROOTFS}")
 
-if("${CLI_CMAKE_RUNTIME_ID}" MATCHES "tizen.*")
```

Comment:
@gkhanna79 @hqueue In fact, I exported `__base_rid` because I wanted to use as much of the existing core-setup code as possible. I think it is better to use a variable declaration without `__` at export, but since it uses `__DistroRid` in CoreCLR or CoreFX, it would be nice to use this variable. What about it ?
CoreCLR:https://github.com/dotnet/coreclr/blob/master/build.sh#L77, CoreFX:https://github.com/dotnet/corefx/blob/master/src/Native/build-native.sh#L46